### PR TITLE
[azure-docs][rfc] docs on setting up AKS agent

### PR DESCRIPTION
**Pull request by dpeng817**
This PR is the first in a stack creating a "zero to one" guide for deploying Dagster on azure infrastructure. The idea is to make Dagster on azure feel more first class and provide a richly linked, linear, guided experience for setting up a deployment. Everything from initial agent setup to CI/CD to compute logs.
## Summary & Motivation
Adds initial docs on setting up an AKS agent. Includes the azure specific incantations necessary to get kubectl working + stand up the helm chart on an AKS cluster (including perms).

I would like someone more knowledgeable than me on K8s best practices to vet what I've written here and make sure we're not walking people into security issues.

### CODE REVIEW

**Comment by dpeng817** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/26186?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#26187** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/26187?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#26186** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/26186?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/26186?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* **#26158** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/26158?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 1 other dependent PR ([#26171](https://github.com/dagster-io/dagster/pull/26171) <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/26171?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* **#26157** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/26157?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `master`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: docs/content/dagster-plus/deployment/azure/aks-agent.mdx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+We should now be able to verify our installation by running a command that tells us the current context of our kubectl installation. We'd expect it to output the name of the AKS cluster.
+
+```bash
+kubectl config current-context
+```
+
+## Step 3: Install the Dagster+ agent on the AKS cluster.
+
+Next, we'll install the agent helm chart. You should be able to follow the guide [here](/concepts/deployment/agents/kubernetes/configuring-running-kubernetes-agent) to install the agent on the AKS cluster.
```
Comment (mlarose) [inline docs/content/dagster-plus/deployment/azure/aks-agent.mdx:40]: <i class='graphite__hidden'>[Re: line +41]<br/><br/></i>

nit: feels a bit dry. maybe something like. 

"Once you've completed this step, you should have a running Dagster+ agent ready to accept your code locations, which you will deploy in (link to CI/CD)" ? <span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/26186?utm_source=unchanged-line-comment">Graphite</a>.</span>

**Review by mlarose — APPROVED** [review]
lgtm

File: docs/content/dagster-plus/deployment/azure/aks-agent.mdx (hunk 1 above)
Comment (mlarose) [reply docs/content/dagster-plus/deployment/azure/aks-agent.mdx:40]: nvm - already done upstack

**Pull request merged**